indra/tests/test_pubchem_client.py

```diff
@@ -4,3 +4,9 @@
 def test_get_inchi_key():
     ik = pubchem_client.get_inchi_key('5280613')
     assert ik == 'JLVSPVFPBBFMBE-HXSWCURESA-O', ik
+
+
+def test_get_pmids():
+    pmids = pubchem_client.get_pmids('2244')
+    example_pmid = '19036898'
+    assert pmids[0] == example_pmid
```

Comment:
I wouldn't do it this way since the order of PMIDs could be non-deterministic or change over time. How about
```
assert example_pmid in pmids
```